src/redux/reducer.js

```diff
@@ -1,43 +1,15 @@
 var combineReducers = require('redux').combineReducers;
 
-var actionTypes = require('./actions.js').types;
-
-
-var sessionReducer = function (state, action) {
-    // Reducer for handling changes to session state
-    if (typeof state === 'undefined') {
-        state = {};
-    }
-    switch (action.type) {
-    case actionTypes.SET_SESSION:
-        return action.session;
-    case actionTypes.SET_SESSION_ERROR:
-        // TODO: do something with action.error
-        return state;
-    default:
-        return state;
-    }
-};
-
-var tokenReducer = function (state, action) {
-    // Reducer for updating the api token
-    if (typeof state === 'undefined') {
-        state = '';
-    }
-    switch (action.type) {
-    case actionTypes.SET_TOKEN:
-        return action.token;
-    case actionTypes.SET_TOKEN_ERROR:
-        // TODO: do something with the error
-        return state;
-    default:
-        return state;
-    }
-};
+var authReducers = require('./auth.js');
+var conferenceScheduleReducers = require('./conference-schedule.js');
+var conferenceDetailsReducers = require('./conference-details.js');
 
 var appReducer = combineReducers({
-    session: sessionReducer,
-    token: tokenReducer
+    session: authReducers.sessionReducer,
+    token: authReducers.tokenReducer,
+    day: conferenceScheduleReducers.dayReducer,
+    schedule: conferenceScheduleReducers.scheduleReducer,
+    details: conferenceDetailsReducers.detailsReducer
```

Comment:
1 reducer per file makes sense for now – I think that it could become unwieldly if we start to have a considerable number of actions that are unique to specific components, but I also guess the idea is that actions are not tightly coupled to components, and can be used across components, right?

I'll split it out further into the separate reducer files as part of this :)